.github/workflows/ci.yml

```
@@ -145,7 +145,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-           go-version: '1.21.3'
+           go-version: '1.21.6'
```

```suggestion
           go-version-file: 'go.mod'
```

This will read the version from go.mod so that we don't have to update in multiple places.